common-practices-tools/security/yubikey.md

```diff
@@ -0,0 +1,309 @@
+-   If you have funky devices/apps that don't support TFA, you can set an application specific password using that tab. This includes sending E-mail from your personal Gmail account using your civicactions.com IMAP, for instance.
+
+### GitHub
+
+-   Visit <https://github.com/settings/two_factor_authentication/configure>
+-   Enable TFA, and complete the phone verification process (phone will act as backup TFA).
+-   Then you can "Register new device" in the "Security keys" section
+
+### AWS Root Account
+
+For each AWS account you have:
+
+-   Visit <https://console.aws.amazon.com/iam/home?region=us-east-1#security_credential>
+-   Under MFA, add a Virtual MFA device.
+-   Use YubiKey Authenticator app to scan the QR code, and enter the response code, then close and reopen the app and enter the second response code.
+
+### AWS IAM Account
+
+Up to 8 different MFA devices can be assigned, with a combination of Yubikeys, hardware TOTP tokens and authenticator apps.
+
+-   Visit <https://console.aws.amazon.com/iam/home?region=us-east-1#users>
+-   Choose your user name
+-   Select the "Security Credentials" tab
+-   Under Multi-factor Authentication(MFA), select "Assign MFA device"
+-   Enter a Device Name and Select "Security Key"
+-   Allow your browser access to your Yubikey, if prompted
+-   Enter your Yubikey's pin, if prompted
+-   Touch your Yubikey when instructed
+
+## YubiKey Neo U2F Setup
+
+Some older YubiKeys (like the Neo) need to have U2F (Universal Second Factor) enabled before use. _(If you are unsure, ask any IT staff member or on the [#general](https://civicactions.slack.com/messages/general) slack channel.)_
+
+For these YubiKeys, you may need to install and configure some software that "personalizes" your YubiKey. _Note: newer YubiKeys may not require this step._
+
+### Linux
+
+This is the GNU/Linux specific documentation for YubiKey.
+
+_Please help make this page more useful by adding links you found useful (describe exactly how they are useful) and specific steps you used to install, configure, and test your YubiKey._
+
+#### Arch
+
+_See also: <https://wiki.archlinux.org/index.php/yubikey>_
+
+```bash
+pacaur -S perl-net-ldap-server    # this is a prerequisite
+pacaur -S yubikey-neo-manager-git
+```
+
+#### Fedora
+
+_See also: <https://fedoraproject.org/wiki/Using_Yubikeys_with_Fedora>_
+
+```bash
+dnf copr enable jjelen/yubikey-neo-manager
+dnf copr enable spartacus06/yubikey-utils
+dnf install yubikey-neo-manager yubioath-desktop yubikey-personalization-gui
+```
+
+#### Ubuntu, Xubuntu
+
+_See also: <https://askubuntu.com/questions/720314/how-to-install-yubikey-personalization-tool-on-ubuntu>_
+
+```bash
+sudo add-apt-repository ppa:yubico/stable
+sudo apt-get update
+sudo apt-get install yubikey-neo-manager yubikey-personalization yubikey-personalization-gui
+```
+
+#### Locking your Machine with YubiKey
```

Comment:
This can become a two-octothorp (##) top-level section

Reply:
This section is specific to Mac or Linux users, so it's a sub-heading under that.